packages/app-desktop/gui/MainScreen/commands/toggleMenuBar.ts

```diff
@@ -0,0 +1,26 @@
+import { CommandDeclaration, CommandRuntime } from '@joplin/lib/services/CommandService';
+import bridge from '../../../services/bridge';
+import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
+import { _ } from '@joplin/lib/locale';
+
+export const declaration: CommandDeclaration = {
+	name: 'toggleMenuBar',
+	label: () => _('Toggle menu bar'),
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async () => {
+			// Defensive code: macOS disallows hiding the menu bar, so ignore this Command
+			if (shim.isMac()) return;
+
+			const electronWindow = bridge().window();
+			Setting.toggle('hideMenuBar');
+
+			// Now apply the setting
+			electronWindow.setAutoHideMenuBar(Setting.value('hideMenuBar'));
+			electronWindow.setMenuBarVisibility(!Setting.value('hideMenuBar'));
```

Comment:
Please remove, since you're already doing this in the reducer middleware

Reply:
I've removed this file and the references to this command in the latest commit